metadata-models/src/main/pegasus/com/linkedin/schema/EditableSchemaFieldInfo.pdl

```diff
@@ -1,60 +1,27 @@
 namespace com.linkedin.schema
-
-import com.linkedin.common.GlobalTags
-import com.linkedin.common.GlossaryTerms
+import com.linkedin.businessattribute.BusinessAttributeAssociation
 
 /**
  * SchemaField to describe metadata related to dataset schema.
  */
-record EditableSchemaFieldInfo {
-  /**
-   * FieldPath uniquely identifying the SchemaField this metadata is associated with
-   */
-  fieldPath: string
-
-  /**
-   * Description
-   */
-  @Searchable = {
-    "fieldName": "editedFieldDescriptions",
-    "fieldType": "TEXT",
-    "boostScore": 0.1
-  }
-  description: optional string
-
-  /**
-   * Tags associated with the field
-   */
-  @Relationship = {
-    "/tags/*/tag": {
-      "name": "EditableSchemaFieldTaggedWith",
-        "entityTypes": [ "tag" ]
-    }
-  }
-  @Searchable = {
-    "/tags/*/tag": {
-      "fieldName": "editedFieldTags",
-      "fieldType": "URN",
-      "boostScore": 0.5
-    }
-  }
-  globalTags: optional GlobalTags
+record EditableSchemaFieldInfo includes EditableSchemaFieldBase {
```

Comment:
what was the reason behind refactoring the previous fields out into a new class?

Reply:
If we associate businessAttribute with SchemaField entity, then this might not be require anymore and this require more indepth analysis and we will get back to you if this is still required.

Reply:
@shirshanka as suggested, EditableSchemaFieldBase now removed